packages/aws-otel-collector/aws-otel-collector-default-config.yaml

```diff
@@ -0,0 +1,37 @@
+extensions:
+  health_check:
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
```

Comment:
In the collector logs, I see the following warning:
```
{"level":"warn","timestamp":"2024-07-24T07:25:10.048Z","caller":"internal/warning.go:42","message":"Using the 0.0.0.0 address exposes this server to every network interface, which may facilitate Denial of Service attacks. Enable the feature gate to change the default and remove this warning.","kind":"extension","name":"health_check","documentation":"https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/security-best-practices.md#safeguards-against-denial-of-service-attacks","feature gate ID":"component.UseLocalHostAsDefaultHost"}
```

so I'm not sure if we should include this setting.

Reply:
Please set these to 127.0.0.1 instead so the services are only accessible to local clients.

Reply:
Ack, will change these endpoints to localhost IP